Mathlib/Algebra/Algebra/Defs.lean

```diff
@@ -311,6 +324,45 @@ section
 
 end
 
+section compHom
+variable {R : Type u} {S : Type v} (A : Type w)
+    [CommSemiring R] [Semiring A] [Algebra R A] [CommSemiring S]
+    (f : S →+* R)
+
+/--
+Compose a Algebra with a RingHom, with action f s • m.
+-/
+abbrev compHom: Algebra S A where
```

Comment:
style nit:
```suggestion
abbrev compHom : Algebra S A where
```
Generally, there should always be spaces around colons; can you go through the PR and fix these throughout?